backend/SlothOrganizer/SlothOrganizer.Services.Tests.Unit/AuthServiceTests.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Authentication;
+using System.Text;
+using System.Threading.Tasks;
+using FakeItEasy;
+using SlothOrganizer.Contracts.DTO.Auth;
+using SlothOrganizer.Contracts.DTO.User;
+using SlothOrganizer.Domain.Exceptions;
+using SlothOrganizer.Services.Abstractions.Auth;
+using SlothOrganizer.Services.Abstractions.Email;
+using SlothOrganizer.Services.Abstractions.Users;
+using SlothOrganizer.Services.Auth;
+using Xunit;
+
+namespace SlothOrganizer.Services.Tests.Unit
+{
+    public class AuthServiceTests
+    {
+        private readonly AuthService _sut;
```

Comment:
_sut - what does it mean? do not use abbreveations withing the application if they are not clear, not required, it is a project domain must-known abbreveations. Same for single letters, magic numbers, etc.

Reply:
Fixed